README.md

```diff
@@ -1,5 +1,25 @@
 WebGL 2 Samples Pack
 =====================
 
-Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://www.linkedin.com/in/shuai-shao-3718818b) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania. 
+GH-Page: http://webglsamples.org/WebGL2Samples/
 
+Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://shrekshao.github.io) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania.
+
+## Samples list
+
+|name          |  port from | Chrome 47 Win | Firefox 42 Win|
+|--------------|------------|:---------------:|:---------------:|
+|draw instanced|  [gl-320-draw-instanced](https://github.com/g-truc/ogl-samples/blob/master/tests/gl-320-draw-instanced.cpp)|   :o:    |  :x: crash        |
+
+
+## Local Test Instructions
+
+* clone the repo
+* You can launch a simple http server by using ```python -m SimpleHTTPServer``` that starts a server at port 8000.
+* If there is no webgl 2.0 support for your browser, there will be a text line at the top of the sample page.
```

Comment:
Got it

I have apply these changes to glsl discard #4

On Thu, Dec 24, 2015 at 9:44 AM, Patrick Cozzi <notifications@github.com>
wrote:

> In README.md
> <https://github.com/WebGLSamples/WebGL2Samples/pull/3#discussion_r48417011>
> :
>
> >
> > +Samples demonstrating WebGL 2 features by [Shuai Shao (Shrek)](https://shrekshao.github.io) and [Trung Le](http://www.trungtuanle.com/). Advised by [Patrick Cozzi](http://www.seas.upenn.edu/~pcozzi/), University of Pennsylvania.
> > +
> > +## Samples list
> > +
> > +|name          |  port from | Chrome 47 Win | Firefox 42 Win|
> > +|--------------|------------|:---------------:|:---------------:|
> > +|draw instanced|  [gl-320-draw-instanced](https://github.com/g-truc/ogl-samples/blob/master/tests/gl-320-draw-instanced.cpp)|   :o:    |  :x: crash        |
> > +
> > +
> > +## Local Test Instructions
> > +
> > +* clone the repo
> > +* You can launch a simple http server by using ```python -m SimpleHTTPServer``` that starts a server at port 8000.
> > +* If there is no webgl 2.0 support for your browser, there will be a text line at the top of the sample page.
>
> I will change some of these, but throughout, it should be "WebGL", not
> "webgl"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WebGLSamples/WebGL2Samples/pull/3/files#r48417011>.
>